composer.json

```diff
@@ -17,7 +17,7 @@
         "source": "https://github.com/matomo-org/matomo-php-tracker"
     },
     "require": {
-        "php": "^7.0 || ^8.0",
+        "php": "^7.2 || ^8.0",
```

Comment:
We may as well keep 7.0, does it cause any issues?

Reply:
@michalkleiner what are the reasons to keep 7.0 and 7.1 versions?) I thought, during the discussion we decided to increase min version up to 7.2. If we would keep support 7.0+, this PR makes no sense.

Reply:
I would flip the question and ask what is the reason to increase the minimum to 7.2?
7.2 was mentioned as the minimum the core supports, but it doesn't mean this library needs to follow that. It makes sense when it runs on the same server, but if it runs elsewhere, the version can be different.

So if we went back to my original feedback, and to the question you arrived now to as well — why do we need to change anything here when we haven't decided to drop any support/prevent installations on old versions of PHP?

The only changes that make sense from my perspective, at this stage, are changes to allow support/installation on newer versions of PHP (8.1, 8.2, 8.3).

Reply:
@michalkleiner 
> why do we need to change anything here when we haven't decided to drop any support/prevent installations on old versions of PHP?

I think this question is an answer)

My idea was to make change more smooth and step by step. Of course, we can create a PR with min version 8.2 and rewrite everything to use new features. But it could be difficult for users to make so serious jump.

So, I will close PR next week if we don't decide that we need these changes.

Reply:
The constants visibility is in PHP from 7.1, so in order to keep that we need to bump the minimal version. Let's stay with 7.2 then, not a huge difference, both EOL, and Matomo already requires 7.2.